### PR TITLE
MLOps job description

### PR DESCRIPTION
**Pull request by vipul-sharma20**

### CODE REVIEW

File: machine-learning/ml-ops.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+authors: vipul-sharma20
+groups: product
+teams: platform-ml
+level: [L3, L4]
+active: true
+---
```
Comment (lepisma) [inline machine-learning/ml-ops.md]: let's add `draft: true` since approval for label + role is not done yet.

File: machine-learning/ml-ops.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- Design and build CT/CI/CD pipelines from ground up for ML
+  life-cycle management.
+- Collaboratively solve complex problems, develop complex deployment and
+  scalable infrastructure as code.
+- Contribute to the development roadmap to build and operationalize our ML
+  environment, platforms and tooling.
+
+### Requirements
+
+- More than x years of experience.
```
Comment (lepisma) [inline machine-learning/ml-ops.md]: 2+ is fine

File: machine-learning/ml-ops.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+---
+authors: vipul-sharma20
+groups: product
+teams: platform-ml
```
Comment (lepisma) [inline machine-learning/ml-ops.md]: let's use `platform` as the team? I am using that for other roles too.

File: machine-learning/ml-ops.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
```
Comment (lepisma) [inline machine-learning/ml-ops.md]: Can we move it to Engineering?

**Review by lepisma — CHANGES_REQUESTED** [review]

**Comment by vipul-sharma20** [comment]
@lepisma Updated the JD with suggested changes.

**Pull request merged**